config/management/src/validator_config.rs

```diff
@@ -87,6 +86,20 @@ impl ValidatorConfig {
                 )
             })?;
 
+        // Import the shared_val_netaddr_key to local storage
```

Comment:
this is being set by terraform, we can also have an explicit command to set it, but we should only be reading it